### PR TITLE
typo: `InlineEnviromentVariablesPlugin` -> `InlineEnvironmentVariablesPlugin`

### PR DESCRIPTION
**Pull request by marian-c**

### CODE REVIEW

**Comment by marian-c** [comment]
just saw the same type in the example file also
[example/webpack.config.js](https://github.com/tikotzky/inline-environment-variables-webpack-plugin/blob/6e203afae5e7c58dddc8c1e874fb6f643bb4f5c8/example/webpack.config.js)

**Comment by tikotzky** [comment]
Thanks, would you like to correct the typo in the example file as well?
If not I can merge this now...

**Comment by marian-c** [comment]
I've fixed the example but also the `index.js` file

File: example/webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // var InlineEnviromentVariablesPlugin = require('inline-environment-variables-webpack-plugin');
```
Comment (tikotzky) [inline example/webpack.config.js]: Can you update this comment as well?

Comment (marian-c) [reply example/webpack.config.js]: :) relied too much on the IDE,
fixed

**Comment by marian-c** [comment]
@tikotzky I've only renamed the vars, I'll also check that it still runs (just to be completely safe)

--
all good, ran the plugin successfully

**Comment by tikotzky** [comment]
👍 awesome thanks!

**Pull request merged**